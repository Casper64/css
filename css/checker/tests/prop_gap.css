
.gap {
    gap: 10px 20px 30px;
    gap: what;
    gap: 20vw;
}