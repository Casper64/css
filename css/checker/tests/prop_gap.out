css/checker/tests/prop_gap.css:3:10: error: invalid value for property "gap"
    1 | 
    2 | .gap {
    3 |     gap: 10px 20px 30px;
      |          ~~~~~~~~~~~~~~~
    4 |     gap: what;
    5 |     gap: 20vw;